### DISCUSSION TITLE
Stream Category

### DISCUSSION THREAD

**Discussion by Tederic** [Cog-Creators/Red-DiscordBot#5374] [Ideas]
Hello there,

I suggest that we add an additional command in the /streamset for Twitch.
We should be able to choose the category / categories for which the bot will publish.

For example :

/streamset twitchcategory New World

Discord servers often specialize in game types and don't necessarily need to see all categories.

It seems that class TwitchGame(Game) is able to detect this information
https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/feature/streams_game_alerts/redbot/cogs/streams/streamtypes.py

**Comment by ltzmax** [comment]
This sounds like [Red#4174](https://github.com/Cog-Creators/Red-DiscordBot/pull/4174). But tbh i do like the idea of it.
